MAUI/DataGrid/column-resizing.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: Column Resizing in .NET MAUI DataGrid control | Syncfusion
+description: Learn all about column reizing support in Syncfusion MAUI DataGrid (SfDataGrid) control and more here.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Resizing columns in MAUI DataGrid (SfDataGrid)
```

Comment:
Columns Resizing in MAUI DataGrid (SfDataGrid)

Reply:
Addressed the review concerns provided.